WordPress/src/main/java/org/wordpress/android/ui/reader/viewmodels/ReaderPostListViewModel.kt

```diff
@@ -244,10 +264,19 @@ class ReaderPostListViewModel @Inject constructor(
         _changeBottomSheetVisibility.value = Event(false)
     }
 
-    fun onSubfilterChanged(
+    private fun changeSubfilter(
         subfilterListItem: SubfilterListItem,
         requestNewerPosts: Boolean
     ) {
+        if (subfilterListItem.isTrackedItem &&
+                            !readerTracker.isRunning(ReaderTrackerType.SUBFILTERED_LIST)) {
+            AppLog.d(T.READER, "TRACK READER ReaderPostListFragment > START Count SUBFILTERED_LIST")
+            readerTracker.start(ReaderTrackerType.SUBFILTERED_LIST)
+        } else if (!subfilterListItem.isTrackedItem && readerTracker.isRunning(ReaderTrackerType.SUBFILTERED_LIST)) {
+            AppLog.d(T.READER, "TRACK READER ReaderPostListFragment > STOP Count SUBFILTERED_LIST")
+            readerTracker.stop(ReaderTrackerType.SUBFILTERED_LIST)
+        }
```

Comment:
My previous comment about the code being a bit hard to read could also be applied here.

Reply:
Same as above [here](56f01bceb31845b0c9a545c26dedf0a39fd75fc7)